_summaries/wu2021rethinking_2.md

```diff
@@ -1,19 +1,90 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: wu2021rethinking
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: aabayomi
+score: 10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+### What is the core idea?
+
+The authors argues using positive random orthogonal features can estimate softmax showing speed and performance gains over SOTA transformer architecture
+
```

Comment:
Can you also add how the paper changes attention computation for quadratic time complexity to linear time complexity?

Reply:
Also, can you add that paper introduced Performer which uses linear space and complexity without depending on priors like low-rankness or sparsity? Also can you talk a bit about FAVOR+.